spec/mini_twitter_client_spec.rb

```diff
@@ -0,0 +1,20 @@
+require './lib/mini_twitter_client'
+require 'debug'
+
+RSpec.describe 'methods of mini_twitter_client class: create_tweet(author, massage) and delete_tweet(id)' do
+  it ' creates new tweet and delete it ' do
+    author = 'Johan'
+    email = 'johan@johan.com'
+    message = 'Test tweet :)'
+
+    def mini_twitter_client
+      MiniTwitterClient.new
+    end
```

Comment:
You can just assign it to variable:
```suggestion
    mini_twitter_client = MiniTwitterClient.new
```

Reply:
Yes, forgot to change :)